### ISSUE TITLE
GTK2/GTK3 button text color in LibreOffice

### ISSUE THREAD

**Issue by thewaiter** [themix-project/oomox-gtk-theme#64]
Hello guys
 
I am using oomox for our Bodhi Linux distribution and I am pretty happy that is an app like this to help me maintain our themes. Anyway I see some troubles with gtk 3.20. Look at the picture. Firefox in the background has fine button text color but Writer/Libre Office does not. Why? Have I missed some setting?
 
https://i.imgur.com/t0wXmpI.jpg
https://i.imgur.com/eG9kZqi.jpg
 
Thanx for your soon reply and excuse me in case I overlooked the same issue which had been responsed.
 
Best regards
 
Stefan

**Comment by actionless** [comment]
yup, i can reproduce the problem with their gtk3 VCL, seems like a problem in that VCL itself

unfortunately gtk3 inspector is not working in libreoffice, so i can't come up with any workaround for this

**Comment by actionless** [comment]
hm, i've actually found a way to force GTK inspector without using the hotkeys:

```
$ env GTK_THEME=oomox-unit_01c GTK_DEBUG=interactive libreoffice
```

however debugger showing some weird output (it just showing whole window as a big single widget instead of showing buttons, input fields and so on):

![2019-09-10--1568126316_3840x2160_scrot](https://user-images.githubusercontent.com/1655669/64623601-8264de00-d3e9-11e9-9365-92491c9b3b06.png)

**Comment by actionless** [comment]
also, just out of curiosity i've tried gtk2 VCL, and it have the same issue (button labels using color of normal labels):

![2019-09-10--1568126807_3536x1123_scrot](https://user-images.githubusercontent.com/1655669/64624269-b096ed80-d3ea-11e9-87d0-3598d97d7651.png)

**Comment by actionless** [comment]
at this point it just feels like a question/bug to report to developers of GTK2 and GTK3 LibreOffice VCLs

**Comment by thewaiter** [comment]
Hello there and thanx for your attention. I probably need to ask those developers :(

**Comment by actionless** [comment]
also if you know some gtk3 themes in which such combination (dark labels on background and light labels on buttons or the opposite) are working fine -- we could try to analyze those themes

**Comment by thewaiter** [comment]
Just forced to use Libre Office without GTK 3. It was simple, just purge libreoffice-gtk3. Anyway the same situation with Thunar and combobox. Look at the picture. Maybe I am wrong but what if oomox i buggy? I am using v 1.11.
https://i.imgur.com/HnjYFfL.jpg

**Comment by actionless** [comment]
with thunar i can't reproduce the issue:

![2019-09-10--1568140742_1079x1227_scrot](https://user-images.githubusercontent.com/1655669/64640806-0f6c5f00-d40b-11e9-8999-0035834c9a26.png)


please double-check with the latest version of oomox theme

**Comment by nana-4** [comment]
I can reproduce the LibreOffice issue even with other theme. It seems LibreOffice reads the text color from `.background`, not `button`.


When I manually set `button { color: red; }` (in Materia):

![image](https://user-images.githubusercontent.com/9556384/64696960-b8ce5600-d4da-11e9-8723-1ba3a8c4f2f5.png)

When I manually set `.background { color: red; }`:

![image](https://user-images.githubusercontent.com/9556384/64697305-840ece80-d4db-11e9-8544-8f1f413a1a0f.png)

This seems to be an upstream LibreOffice bug.
